tests/plugins/filters.test.ts

```diff
@@ -186,6 +187,21 @@ describe("Filters plugin", () => {
       expect(model.getters.isRowHidden(sheetId, 1)).toBe(true);
       expect(model.getters.isRowHidden(sheetId, 2)).toBe(false);
     });
+
+    test("Filtered rows should persist after hiding and unhiding multiple rows", () => {
+      const model = new Model({ sheets: [{ colNumber: 8, rowNumber: 8 }] });
+
+      setCellContent(model, "A4", "D");
+      setCellContent(model, "A5", "D");
+      setCellContent(model, "A6", "D");
```

Comment:
the test could be more minimalist. You only need 1 cell to show the behavior, not 3
The smaller the test, the better is it